src/content/changes/highlights/2022-09-08.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2022-09-08T10:00:00
+title: Highlights for the week ending September 8, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+- All management-cluster Flux installations (`flux-system`, `flux-giantswarm`) have been upgraded to Flux [v0.33.0](https://github.com/fluxcd/flux2/releases/tag/v0.33.0).
+
+## User interfaces
+
+- We have created the first [v0.1.0 release](https://github.com/giantswarm/gitops-template/releases/tag/v0.1.0) of our [gitops-template repo](https://github.com/giantswarm/gitops-template/). It includes now all examples showing how to use our GitOps offering to achieve functionality such as management of Organizations, Workload Clusters, Secrets encryption, preparing and using app and cluster templates, environments propagation. To make it easier to use, we're also introducing [a new set of commands to our `kubectl gs` plugin](https://docs.giantswarm.io/ui-api/kubectl-gs/gitops/) that will allow you to easily generate parts of the GitOps configuration instead of just copying and changing it.
```

Comment:
No need to capitalize these words

```suggestion
- We have created the first [v0.1.0 release](https://github.com/giantswarm/gitops-template/releases/tag/v0.1.0) of our [gitops-template repo](https://github.com/giantswarm/gitops-template/). It includes now all examples showing how to use our GitOps offering to achieve functionality such as management of organizations, workload clusters, secrets encryption, preparing and using app and cluster templates, environments propagation. To make it easier to use, we're also introducing [a new set of commands to our `kubectl gs` plugin](https://docs.giantswarm.io/ui-api/kubectl-gs/gitops/) that will allow you to easily generate parts of the GitOps configuration instead of just copying and changing it.
```